modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/service/test/SegmentsExperimentLocalServiceTest.java

```diff
@@ -719,19 +726,28 @@ public void testUpdateSegmentsExperimentStatusToCompletedWithWinnerSegmentsExper
 		segmentsExperience1 =
 			_segmentsExperienceLocalService.fetchSegmentsExperience(
 				segmentsExperience1.getSegmentsExperienceId());
+
+		Assert.assertFalse(segmentsExperience1.isActive());
+		Assert.assertEquals(
+			priorityVariantSegmentsExperience,
+			segmentsExperience1.getPriority());
+
 		segmentsExperience2 =
 			_segmentsExperienceLocalService.fetchSegmentsExperience(
 				segmentsExperience2.getSegmentsExperienceId());
 
-		Assert.assertFalse(segmentsExperience1.isActive());
-
 		Assert.assertTrue(segmentsExperience2.isActive());
+		Assert.assertEquals(
+			prioritySegmentsExperience2, segmentsExperience2.getPriority());
 
 		variantSegmentsExperience =
 			_segmentsExperienceLocalService.fetchSegmentsExperience(
 				variantSegmentsExperience.getSegmentsExperienceId());
 
 		Assert.assertTrue(variantSegmentsExperience.isActive());
+		Assert.assertEquals(
+			prioritySegmentsExperience1,
```

Comment:
Better if we use a real value, it is really hard to now what is the expected value here.

Reply:
I like it, applying ;)